src/App.jsx

```diff
@@ -13,6 +15,8 @@ const App = () => {
         <Routes>
           <Route path="/" element={<RecipesMainView />} />
           <Route path="recipes/:id" element={<RecipeFullView />} />
+          <Route path="registration-form" element={<RegistrationForm />} />
```

Comment:
"registration-form" as a path it doesn't sound nice...
I would use something like "signup", cleaner and more concise.

Reply:
Followed advice and changed.